scripts/run_test.py

```diff
@@ -279,6 +275,10 @@
     memservice_config_doc["ATL_queue_size"] = args.atlqsize
 if args.atldatasize is not None:
     memservice_config_doc["ATL_data_size"] = args.atldatasize
+    memservice_config_doc["Memservers"] = args.memservers.split("/")
```

Comment:
If we enable multi memory server test in build_and_test.sh - I get below error : 

Error code: 4
Error msg: get_config_info:547:memory_type option in the config file is invalid.